DrawBot.py

```diff
@@ -248,5 +260,26 @@ def application_openFile_(self, app, path):
         return False
 
 
+def _getPIPTargetPath():
+    appSupportPath = AppKit.NSSearchPathForDirectoriesInDomains(
+            AppKit.NSApplicationSupportDirectory,
+            AppKit.NSUserDomainMask, True)[0]
+    version = f"{sys.version_info.major}.{sys.version_info.minor}"
+    return os.path.join(appSupportPath, f"DrawBot/Python{version}")
+
+
+def _addLocalSysPaths():
+    version = f"{sys.version_info.major}.{sys.version_info.minor}"
+    paths = [
+        _getPIPTargetPath(),
+        '/Library/Python/%s/site-packages' % version,
+    ]
+    for path in paths:
+        if path not in sys.path and os.path.exists(path):
+            site.addsitedir(path)
+
+_addLocalSysPaths()
```

Comment:
this has to be called before all imports otherwise is not possible to overwrite embedded packages